projects/golang/go/1.17/patches/0005-go1.17.13-eks-os-net-http-avoid-escapes-from-os.DirF.patch

```diff
@@ -0,0 +1,513 @@
+From 21d77ea72e5023d7df5769ee97430ae326009639 Mon Sep 17 00:00:00 2001
+From: Damien Neil <dneil@google.com>
+Date: Thu, 10 Nov 2022 12:16:27 -0800
+Subject: [PATCH 5/9] [go1.17.13-eks] os, net/http: avoid escapes from os.DirFS
+ and http.Dir on Windows
+
```

Comment:
let's add the standard header we've been using for these patch files here as well

Reply:
see https://github.com/aws/eks-distro-build-tooling/tree/main/projects/golang/go for an example